.circleci/config.yml

```diff
@@ -792,6 +792,11 @@ jobs:
       - store_artifacts:
           path: development/ts-migration-dashboard/build
           destination: ts-migration-dashboard
+      - run:
+          name: Set branch parent commit env var
+          command: |
+            echo "export PARENT_COMMIT=$(git rev-parse origin/HEAD)" >> $BASH_ENV
+            source $BASH_ENV
```

Comment:
Hmm, this is a confusing way to do this. Source means "run this in the current shell". We can run it in the current shell directly, rather than putting the command into a string and sourcing it.

Reply:
Though.... I suspect we don't need to do this at all. CircleCI has the base commit, and exposes it as `pipeline.git.base_revision`. See https://circleci.com/docs/variables/#built-in-environment-variables

This can be set as an environment variable (see the example in that section)

Reply:
Hi @Gudahtt, yes I already searched around for circleci variables but as you found out there were some reports of shenanigans around `pipeline.git.base_revision`.

I'll apply your suggestion on the bash script anyway!

Reply:
I'm sorry there was a mix up here; my last comment was meant as a dismissal of my entire suggestion. The change I suggested was wrong because CircleCI requires that we use `source` when setting environment variables, so that they can set them for later jobs. Sorry about that! I guess we'll see if this run passes, but I expect you'll need to bring back the original pattern of

```
echo "..." >> $BASH_ENV
source $BASH_ENV
```

Reply:
My bad! I missed your last comment 😄  
I rolled back to the original `source` pattern, as the env var was undefined as expected in the last job run